### DISCUSSION TITLE
New Roads on Mapbox

### DISCUSSION THREAD

**Discussion by sani-0401** [rnmapbox/maps#3259] [Q&A]
I have been working on a project, where I need to have navigation functionality on the property. But the internal roads on the property are not displayed on the mapbox.
As the mapbox uses OSM as base layer. Do I need to add the roads in the OSM to reflect the roads in mapbox.
what could be the solution for this.

**Comment by RyanTG** [comment]
Yes, as far as I know adding the data to OSM is the correct path to take. However, I'm not sure how often mapbox integrates OSM data into their tilesets, so there may be a delay before you see your changes.
